tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -294,3 +294,5 @@ def test_get_metrics(
     ) == deep_filter_properties_iterable(
         properties=expected_rule_based_profiler_config.to_json_dict()
     )
+
+    assert 0.0 < result.execution_time <= 1.0
```

Comment:
Could we add an inline comment here saying that `execution_time` is seconds?

Reply:
@Shinnnyshinshin Done!  Thanks.